### PR TITLE
Feat: Next 13 - Global Section - Footer Section

### PR DESCRIPTION
**Pull request by lariciamota**
## What's the purpose of this pull request?

Migration of the `Footer` section from the pages router to the app router (Next 13).

## How it works?
Screenshot 2024-06-11 at 16 16 02

## How to test it?

- Run `yarn dev`
- Make sure that all the current pages are working as expected (`/`, `/office` and some `/<slug>/p`);
- Make sure `/fs-next-update` is rendering with the `Footer` section on the bottom of the page and without errors.

## References

- [Jira task](https://vtex-dev.atlassian.net/browse/SFS-1020)

### CODE REVIEW

File: packages/core/app/components/sections/Footer/Footer.tsx [hunk 1]
```diff
@@ -14,8 +14,8 @@ import type { FooterLinksProps, FooterSocialProps } from '../../common/Footer'
 
 import Logo from 'src/components/ui/Logo'
 import Link from 'src/components/ui/Link'
-import UIIncentives from '../../ui/Incentives'
-import type { Incentive } from '../../ui/Incentives'
+import UIIncentives from '../../../../src/components/ui/Incentives'
+import type { Incentive } from '../../../../src/components/ui/Incentives'
```
Comment (eduardoformiga) [inline packages/core/app/components/sections/Footer/Footer.tsx:18]: Do you think it is worth migrating these internal components now as well?

Comment (lariciamota) [reply packages/core/app/components/sections/Footer/Footer.tsx]: Sure, no problem

File: packages/core/app/components/sections/Footer/Footer.tsx
Comment (eduardoformiga) [inline packages/core/app/components/sections/Footer/Footer.tsx:1]: from the PR 
- https://github.com/vtex/faststore/pull/2347


we are going to migrate the Logo and Link as well. We need to sync the order of the merges and sync appropriately 😉 
Screenshot 2024-06-13 at 17 41 57

**Review by eduardoformiga — APPROVED** [review]
LGTM!

**Pull request merged**